ledger/participant-integration-api/src/main/resources/db/migration/postgres/V50__Append_only_schema.sql

```diff
@@ -121,52 +121,541 @@ CREATE TABLE participant_events_divulgence             PARTITION OF participant_
+command_id,
+workflow_id,
+application_id,
+submitters,
+node_index,
+event_id,
+contract_id,
+template_id,
+flat_event_witnesses,
+tree_event_witnesses,
+create_argument,
+create_signatories,
+create_observers,
+create_agreement_text,
+create_key_value,
+create_key_hash,
+exercise_choice,
+exercise_argument,
+exercise_result,
+exercise_actors,
+exercise_child_event_ids,
+create_argument_compression,
+create_key_value_compression,
+exercise_argument_compression,
+exercise_result_compression
+*/
+
+-- TODO append-only: EITHER only include columns that are used in queries that use this view OR verify that the query planning
+-- is not negatively affected by a long list of columns that are never used.
+CREATE VIEW participant_events
+    AS
+        SELECT
+            event_kind,
+            event_sequential_id,
+            NULL as event_offset,
+            NULL as transaction_id,
+            NULL as ledger_effective_time,
+            command_id,
+            workflow_id,
+            application_id,
+            submitters,
+            NULL as node_index,
+            NULL as event_id,
+            contract_id,
+            template_id,
+            NULL as flat_event_witnesses,
+            tree_event_witnesses,
+            create_argument,
+            NULL as create_signatories,
+            NULL as create_observers,
+            NULL as create_agreement_text,
+            NULL as create_key_value,
+            NULL as create_key_hash,
+            NULL as exercise_choice,
+            NULL as exercise_argument,
+            NULL as exercise_result,
+            NULL as exercise_actors,
+            NULL as exercise_child_event_ids,
+            create_argument_compression,
+            NULL as create_key_value_compression,
+            NULL as exercise_argument_compression,
+            NULL as exercise_result_compression
+        FROM participant_events_divulgence
+    UNION
+        SELECT
+            event_kind,
+            event_sequential_id,
+            event_offset,
+            transaction_id,
+            ledger_effective_time,
+            command_id,
+            workflow_id,
+            application_id,
+            submitters,
+            node_index,
+            event_id,
+            contract_id,
+            template_id,
+            flat_event_witnesses,
+            tree_event_witnesses,
+            create_argument,
+            create_signatories,
+            create_observers,
+            create_agreement_text,
+            create_key_value,
+            create_key_hash,
+            NULL as exercise_choice,
+            NULL as exercise_argument,
+            NULL as exercise_result,
+            NULL as exercise_actors,
+            NULL as exercise_child_event_ids,
+            create_argument_compression,
+            create_key_value_compression,
+            NULL as exercise_argument_compression,
+            NULL as exercise_result_compression
+        FROM participant_events_create
+    UNION
```

Comment:
I believe we need a 'UNION ALL' as 'UNION' does sorting and removal of duplicates.

Reply:
:+1: